requirements-dev.txt

```
@@ -6,8 +6,8 @@ git+https://github.com/astropy/asdf-astropy
 git+https://github.com/spacetelescope/crds
 git+https://github.com/spacetelescope/gwcs
 numpy>=0.0.dev0
-git+https://github.com/spacetelescope/roman_datamodels@0.17.0
-git+https://github.com/spacetelescope/rad@0.17.0
+git+https://github.com/spacetelescope/roman_datamodels@0.17.1
+git+https://github.com/spacetelescope/rad@0.17.1
```

```suggestion
git+https://github.com/spacetelescope/roman_datamodels
git+https://github.com/spacetelescope/rad
```
These should not be pinned to a specific versions. The devdeps tests are supposed to test against the development versions of packages not some specific release.